app/views.py

```diff
@@ -89,11 +88,10 @@ def json_serial(obj):
     raise TypeError("Type %s not serializable" % type(obj))
 
 
-@csrf_exempt
```

Comment:
why was it disabled?

Reply:
csrf uses 2 token for validation.
- token as a hidden form field
- token as a browser cookie
but when i send request (submit form) to our reverse proxy browser do not add cookies to this request. It was a workaround for POC.

Reply:
got it.